proposal/comp110_architecture.bib

```diff
@@ -7,3 +7,91 @@ @article{shannon
 	number={3}, 
 	pages={379-423}
 }
+
+@ARTICLE{smith:launchpad, 
+author={Smith, G. and Whitehead, J. and Mateas, M. and Treanor, M. and March, J. and Mee Cha}, 
+journal={IEEE Transactions on Computational Intelligence and AI in Games}, 
+title={Launchpad: A Rhythm-Based Level Generator for 2-D Platformers}, 
+year={2011}, 
+volume={3}, 
+number={1}, 
+pages={1-16}, 
+month={March},
+}
+
+@ARTICLE{compton:platform, 
+author={Compton, Kate and Mateas, Michael}, 
+booktitle={Proceedings of the Second Artificial Intelligence and Interactive Digital Entertainment International Conference (AIIDE)}, 
+title={Procedural Level Design for Platform Games}, 
+address={Marina del Rey, CA},
+year={2006}, 
+pages={109-111}, 
+month={June},
+}
+
+
+@INPROCEEDINGS{mawhorter:occupancy, 
+author={Mawhorter, P. and Mateas, M.}, 
+booktitle={IEEE Symposium on Computational Intelligence and Games (CIG)}, 
+title={Procedural level generation using occupancy-regulated extension}, 
+year={2010}, 
+pages={351-358},
+month={Aug},
+}
+
+@inproceedings{mourato:genetic,
+ author = {Mourato, Fausto and dos Santos, Manuel Pr\'{o}spero and Birra, Fernando},
```

Comment:
An algorithm is presented and would be suitable for comparison. :+1: Further to this, please note that Mourato has published more recent work in the Interactive Journal of Creative Interfaces and Computer Graphics which you may find useful to draw upon.